src/openforms/authentication/api/views.py

```diff
@@ -12,6 +12,7 @@
 from openforms.submissions.utils import remove_submission_from_session
 from openforms.utils.api.views import ListMixin
 
+from ...api.authentication import AnonCSRFSessionAuthentication
```

Comment:
```suggestion
from openforms.api.authentication import AnonCSRFSessionAuthentication
```
really need to set up a linter/CI check for this :grimacing: 

this is a different "django app", so for those we use absolute imports

Reply:
Some linting/autofixing would be handy, I can't configure PyCharm's auto-import to make the distinction so these keep showing up everywhere.